client/www/components/dash/explorer/EditRowDialog.tsx

```diff
@@ -284,6 +284,7 @@ export function EditRowDialog({
                     tabIndex={tabIndex}
                     value={value}
                     options={[
+                      { value: '', label: '-' },
```

Comment:
This lets us distinguish between `false` and `null`. Without this change you can't set a value to `false` from the edit row dialog.

Reply:
Nice!